visualization/scatter_plot.js

```diff
@@ -60,14 +60,20 @@ const updateXAxis = () => {
 	svg.select('.x-axis').call(xAxis);
 };
 
-const updateScatterPlotYValues = (checkedAttributes) => {
-	normalizeSelectedAttributes(shownPlayers, checkedAttributes);
+const updateScatterPlotYValues = (sliderAttributes) => {
+	normalizeSelectedAttributes(shownPlayers, sliderAttributes);
 	yValue = function(d) {
 		// Calculate the combined score of each of the selected statistics
 		let combinedScore = 0;
-		for (let i = 0; i < checkedAttributes.length; i++) {
-			combinedScore += parseFloat(d[checkedAttributes[i] + '-Normalized']);
+		let norm_value = 0;
+		let weight_total = 0
+		for (var attr in sliderAttributes) {
+			norm_value = parseFloat(d[attr + '-Normalized'])
+			weight_value = parseFloat(sliderAttributes[attr]) / 100

```

Comment:
This is to limit our `weight_value` between 0 and 1 correct?

Reply:
Yep, slider value range is 1-100